src/models/config.py

```diff
@@ -27,3 +24,6 @@ def families(self):
         default = ''
         self.data.setdefault('families', default)
         return self.data['families']
+
+    def command_exists(self):
+        return 'command' in self.data
```

Comment:
Could we make this return a `boolean`?

Reply:
This does return a `boolean` - in this context `x in y` returns true if `x` is a key in dict `y`, false otherwise
It doesn't return the value of command
Am I alright to merge this through then?

Reply:
What happens then if I define my `yaml` as:
```
command:
some-other-key: value
```
Does `x in y` return true or false?

Reply:
Good catch! `yaml` does create a key in data even for an empty field so the previous check was insufficient. My bad, I should've tested that eventuality, apologies. 
https://github.com/alces-software/adminware/pull/88/commits/f0c33d79c8358badb5b1d77c420f9e8cef5e41c2 how's this?